lib/tasks/evm_dba.rake

```diff
@@ -172,6 +172,7 @@ namespace :evm do
           opt :uri,              "Destination depot URI",       :type => :string, :required => true
           opt :uri_username,     "Destination depot username",  :type => :string
           opt :uri_password,     "Destination depot password",  :type => :string
+          opt :uri_region,       "Destination depot region",    :type => :string
```

Comment:
I don't think the AWS region has anything to do with the URI, right? Can this option be called `aws_region` instead?